spec/implementations/pto_next_week/fetch_next_week_pto_from_drive_spec.rb

```
@@ -0,0 +1,62 @@
+      credentials_path: 'spec/fixtures/fake_credentials.json'
+    }
+
+    allow(File).to receive(:open).with('spec/fixtures/fake_credentials.json').and_return(StringIO.new('{}'))
+
+    fake_credentials = instance_double(Google::Auth::ServiceAccountCredentials)
+    allow(Google::Auth::ServiceAccountCredentials).to receive(:make_creds).and_return(fake_credentials)
+    allow(fake_credentials).to receive(:fetch_access_token!).and_return({})
+
+    fake_service = instance_double(Google::Apis::SheetsV4::SheetsService)
+    fake_response = instance_double(Google::Apis::SheetsV4::ValueRange, values: [
+      ['1', 'Laura', '', '07/01/2025', '07/03/2025', '', '', '🏖️', '', 'active']
```

RuboCop (1.75.5)

[convention] Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.

(Layout/FirstArrayElementIndentation)